auth_root_page.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <title>Philanthropy Data Commons</title>
+</head>
+<body>
+  <h1>Philanthropy Data Commons</h1>
+  <p>There is no content here. You probably meant to visit one of these sites:
```

Comment:
If this page were not here, it would be an nginx welcome page. The point remains, though, I agree on everything you said. I created https://github.com/PhilanthropyDataCommons/deploy/issues/69 to track this issue.